src/utils/transactions/getTransactionLink.ts

```diff
@@ -0,0 +1,5 @@
+import { NetworkType } from 'types';
+
+export function getTransactionLink(network: NetworkType, address: string) {
```

Comment:
Not sure we need to pass in the whole network object, we could just pass in the explorerAddress